### DISCUSSION TITLE
Compilation Error occurs in WebDAV

### DISCUSSION THREAD

**Discussion by ttwelevee** [glenn2223/vscode-live-sass-compiler#254] [Q&A] [answered]
I get a Compilation Error when using WebDAV.
Is there a workaround?

Compilation Error
Error: FormatException: Invalid character (at character 15)
aaaaaa.bbb.com@SSL@5050

**Comment by glenn2223** [comment]
Can I see the line of SASS? Just for context

**Comment by ttwelevee** [comment]
This is not a SASS code
I am getting an error for WebDAV filepath.

https://aaaaaa.bbb.com:5050/public_html

filepath in windows
network>aaaaaa.bbb.com:5050>root>public_html

Compilation Error
Error: FormatException: Invalid character (at character 15)
aaaaaa.bbb.com@SSL@5050

**Reply by glenn2223** [reply]
>This is not a SASS code

The only point it shows `Compilation Error` is after attempting to compile a SASS file. I'm not aware of any issues with connecting to network drives.

---

If it's an actual server, it may help to look at [Visual Studio Code Server](https://code.visualstudio.com/docs/remote/vscode-server) - though I'm not too familiar with it myself. However, as I said, it only throws that error after trying to compile some SASS.

**Reply by ttwelevee** [reply]
The problem was solved by switching to SSH connection instead of WebDAV file browsing.
Thank you very much.

**Reply by glenn2223** [reply]
Glad you solved the issue 😁

**Comment by glenn2223** [comment, marked as answer]
As per https://github.com/glenn2223/vscode-live-sass-compiler/discussions/254#discussioncomment-4713658

Switching to a SSH connection fixed the issue
